### PR TITLE
Update cassandra and ALB chart repo to use latest migrated repository

### PR DESCRIPTION
**Pull request by MadhuriArugula**
Travis builds are failing with https://travis-ci.org/github/pegasystems/pega-helm-charts/builds/751186081 chart not found in corresponding repo issues.

Downloading traefik from repo https://charts.helm.sh/stable
Save error occurred: could not find : chart aws-alb-ingress-controller not found in https://kubernetes-charts-incubator.storage.googleapis.com/

Deleting newly downloaded charts, restoring pre-update state

Error: could not find : chart aws-alb-ingress-controller not found in https://kubernetes-charts-incubator.storage.googleapis.com/

Makefile:2: recipe for target 'dependencies' failed

make: *** [dependencies] Error 1

The command "make dependencies" exited with 2.

To Reproduce:
Perform helm dep update and check if dependencies are downloaded properly using "helm dep list" under addons

Expected behavior
All chart dependencies should be downloaded properly without any issues.
The issue is because stable, incubator chart repo used in our charts are deprecated. It should be moved to another repo as per https://helm.sh/blog/new-location-stable-incubator-charts/

References - https://kubernetes-sigs.github.io/aws-load-balancer-controller/guide/upgrade/migrate_v1_v2/

Updated ALB chart and Cassandra chart repositories.

### CODE REVIEW

File: charts/addons/values.yaml [hunk 1]
```diff
@@ -33,24 +33,24 @@ traefik:
...
   enabled: false
   ## Resources created by the ALB Ingress controller will be prefixed with this string
   clusterName: "YOUR_EKS_CLUSTER_NAME"
   ## Auto Discover awsRegion from ec2metadata, set this to true and omit awsRegion when ec2metadata is available.
   autoDiscoverAwsRegion: false
   ## AWS region of k8s cluster, required if ec2metadata is unavailable from controller pod
   ## Required if autoDiscoverAwsRegion != true
-  awsRegion: "YOUR_EKS_CLUSTER_REGION"
-  ## Auto Discover awsVpcID from ec2metadata, set this to true and omit awsVpcID: " when ec2metadata is available.
-  autoDiscoverAwsVpcID: false
```
Comment (kishorv10) [inline charts/addons/values.yaml:47]: Is this stopped supporting in latest chart?

File: terratest/src/test/addons/alb-ingress_test.go [hunk 2]
```diff
@@ -57,110 +57,70 @@ func Test_checkSetAwsRegion(t *testing.T) {
...
-	require.Equal(t, "AWS_ACCESS_KEY_ID", deployment.Spec.Template.Spec.Containers[0].Env[0].Name)
-	require.Equal(t, "YOUR_AWS_ACCESS_KEY_ID", deployment.Spec.Template.Spec.Containers[0].Env[0].Value)
-	require.Equal(t, "AWS_SECRET_ACCESS_KEY", deployment.Spec.Template.Spec.Containers[0].Env[1].Name)
-	require.Equal(t, "YOUR_AWS_SECRET_ACCESS_KEY", deployment.Spec.Template.Spec.Containers[0].Env[1].Value)
-}
-
 var albIngressResources = []SearchResourceOption{
 	{
-		Name: "pega-aws-alb-ingress-controller",
+		Name: "pega-aws-load-balancer-controller",
```
Comment (kishorv10) [inline terratest/src/test/addons/alb-ingress_test.go:111]: Can you add assertion for the SA annotation?

File: charts/addons/values.yaml (hunk 1 above)
Comment (MadhuriArugula) [reply charts/addons/values.yaml:47]: yes, it is not supported/used - https://github.com/aws/eks-charts/blob/master/stable/aws-load-balancer-controller/values.yaml
I just realized autoDiscoverAwsRegion is also not available in the latest chart, I'll remove that as well.

Comment (taz-pega-work) [reply charts/addons/values.yaml:47]: hi - when you update the values file, can you please up date the annotation to something like this?

# When deploying on AWS EKS, set the parameters, install aws-alb-ingress-controller and metrics-server
# to true and then fill in the remaining parameters with your EKS environment details.

please let me know when you are ready and I can review the doc for values.yaml and the readme.md for addons ;-)

Comment (MadhuriArugula) [reply charts/addons/values.yaml:47]: Sure @taz-mon I've updated README and addons values.yaml. Please review.

File: terratest/src/test/addons/alb-ingress_test.go (hunk 2 above)
Comment (MadhuriArugula) [reply terratest/src/test/addons/alb-ingress_test.go:109]: done

**Review by NekhilKotha — APPROVED** [review]

**Review by Heshma — APPROVED** [review]

**Review by taz-pega-work — APPROVED** [review]
excellent!

**Pull request merged**